src/util/manifest.js

```diff
@@ -31,6 +31,12 @@ const manifest = {
     path: 'coursereserves/copyrightstatuses?limit=500',
     shouldRefresh: () => false,
   },
+  displaySettings: {
+    type: 'okapi',
+    // This path is based on the props provided to `ConfigManager` in `DisplaySettings`
+    path: 'configurations/entries?query=(module=COURSES and configName=display)',
+    shouldRefresh: () => false,
+  }
 };
 
 export default manifest;
```

Comment:
This is where I stopped reviewing, as now we go into tapes. I am guessing there is no more actual code buried down among them, but if there is, then I missed it.